### PR TITLE
Make stable validate_lvm_raid1 reboot

### PR DESCRIPTION
**Pull request by manfredi**
- Description: [sporadic] make stable validate_lvm_raid1 reboot
- Related ticket: [poo#154753](https://progress.opensuse.org/issues/154753)
- Verification run: [overview](https://openqa.suse.de/tests/overview?build=issues-154753)

### CODE REVIEW

**Review by rakoenig — COMMENTED** [review]
The title of the commit is confusing and doesn't seem related to the code change. 
Besides that it looks good.

**Comment by jknphy** [comment]
> The title of the commit is confusing and doesn't seem related to the code change. Besides that it looks good.

it looks good as a draft, because at least we are trying to narrow the problem, but VR still are still not stable. Commit's first line and title should (most of the time) match and be descriptive in few characters. At the moment the commit is too short for example (to have some meaning to someone using `git log`.

**Review by sofiasyria — COMMENTED** [review]
We could completely remove line 28. The rebooting and rechecking also might not be necessary. I think just validation of expected disks after first boot should suffice.

**Comment by manfredi** [comment]
Removed the latest calls:
```perl
    _remove_raid_disk($config, $expected_num_devs);
    _reboot();
    $self->wait_boot;
    _check_raid_disks_after_reboot($config, $expected_num_devs);

```

**Review by sofiasyria — APPROVED** [review]
LGTM

**Review by b10n1k — APPROVED** [review]
lgtm

**Comment by lemon-suse** [comment]
Remove unneeded steps make sense. Just a minor comments for the description for the test, only a summary while no test steps, not very friendly for reader. Maybe that's already out of the scope of your ticket, we can discuss such things later.

**Comment by manfredi** [comment]
> Remove unneeded steps make sense. Just a minor comments for the description for the test, only a summary while no test steps, not very friendly for reader. Maybe that's already out of the scope of your ticket, we can discuss such things later.

@lemon-suse  the motivations are explained in the [poo#154753](https://progress.opensuse.org/issues/154753#note-16) comments

File: tests/console/validate_lvm_raid1.pm [hunk 1]
```diff
@@ -54,45 +50,4 @@ sub _check_raid1_partitioning {
...
-
-    assert_script_run 'mdadm --manage ' . $config->{raid1}->{name} . ' --remove ' . $config->{raid1}->{disk_to_fail};
-    script_retry "mdadm --detail " . $config->{raid1}->{name} . " | grep 'Active Devices : $expected_num_devs'";
-    my $removedDisk = script_output "mdadm --detail " . $config->{raid1}->{name} . " | awk '{ if((\$4 == " . $expected_num_devs . ") && (\$5 ~ /removed/)) {print}}'";
-    assert_not_null($removedDisk, "$removedDisk should have been removed");
-}
-
-sub _reboot {
-    record_info('system reboots');
-    power_action('reboot', textmode => 'textmode');
```
Comment (jknphy) [inline tests/console/validate_lvm_raid1.pm:72]: please remove the import on top.

**Comment by lemon-suse** [comment]
> > Remove unneeded steps make sense. Just a minor comments for the description for the test, only a summary while no test steps, not very friendly for reader. Maybe that's already out of the scope of your ticket, we can discuss such things later.
> 
> @lemon-suse the motivations are explained in the [poo#154753](https://progress.opensuse.org/issues/154753#note-16) comments

In fact, my comments only related with the test module that no test steps in description, and this is not only for this test module. Your PR looks good to me. I mean maybe later we can raise a new ticket to update test modules for test steps in description.

**Review by jknphy — APPROVED** [review]

**Review by lemon-suse — APPROVED** [review]

**Comment by jknphy** [comment]
@lemon-suse besides the approval (before merging) please ensure that all the comments are addressed with answers or tickets.

**Pull request merged**